check/Setup.hs

```diff
@@ -0,0 +1,2 @@
+import Distribution.Simple
+main = defaultMain
```

Comment:
I like `Setup.lhs` as they can have a shebang and at the same time be a valid Haskell module.  I would even consider to symlink to `../Setup.lhs` (git can handle symlinks.)